templates/vue/src/components/node-modal/content-form/RichTextForm.vue

```diff
@@ -0,0 +1,288 @@
+    <editor-content
+      class="editor__content"
+      :editor="editor"
+      :value="value"
+      @input="$emit('input', $event.target.value)"
+    />
+  </div>
+</template>
+
+<script>
+import Icon from "./Icon"
+import { Editor, EditorContent, EditorMenuBar } from "tiptap"
+import {
+  Blockquote,
+  CodeBlock,
+  HardBreak,
+  Heading,
+  HorizontalRule,
+  OrderedList,
+  BulletList,
+  ListItem,
+  TodoItem,
+  TodoList,
+  Bold,
+  Code,
+  Italic,
+  Link,
+  Strike,
+  Underline,
+  History,
+  Placeholder,
+} from "tiptap-extensions"
+
+export default {
+  components: {
+    EditorContent,
+    EditorMenuBar,
+    Icon,
+  },
+  props: {
+    value: {
+      type: String,
+      required: true,
+    },
+    placeholder: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      editor: null,
+      icons_first_group: [
+        {
+          vclass: "{ 'is-active': isActive.bold() }",
+          click: "commands.bold",
+          name: "bold",
+          text: "",
+        },
+        {
+          vclass: "{ 'is-active': isActive.italic() }",
+          click: "commands.italic",
+          name: "italic",
+          text: "",
+        },
+        {
+          vclass: "{ 'is-active': isActive.strike() }",
+          click: "commands.strike",
+          name: "strike",
+          text: "",
+        },
+        {
+          vclass: "{ 'is-active': isActive.underline() }",
+          click: "commands.underline",
+          name: "underline",
+          text: "",
+        },
+        {
+          vclass: "{ 'is-active': isActive.code() }",
+          click: "commands.code",
+          name: "code",
+          text: "",
+        },
+        {
+          vclass: "{ 'is-active': isActive.paragraph() }",
+          click: "commands.paragraph",
+          name: "paragraph",
+          text: "",
+        },
+        {
+          vclass: "{ 'is-active': isActive.heading({ level: 2 }) }",
+          click: "commands.heading({ level: 2 })",
+          name: false,
+          text: "H1",
+        },
+        {
+          vclass: "{ 'is-active': isActive.heading({ level: 3 }) }",
+          click: "commands.heading({ level: 3 })",
+          name: false,
+          text: "H2",
+        },
+        {
+          vclass: "{ 'is-active': isActive.heading({ level: 4 }) }",
+          click: "commands.heading({ level: 4 })",
+          name: false,
+          text: "H3",
+        },
+        {
+          vclass: "{ 'is-active': isActive.bullet_list() }",
+          click: "commands.bullet_list",
+          name: "ul",
+          text: "",
+        },
+        {
+          vclass: "{ 'is-active': isActive.ordered_list() }",
+          click: "commands.ordered_list",
+          name: "ol",
+          text: "",
+        },
+        {
+          vclass: "{ 'is-active': isActive.blockquote() }",
+          click: "commands.blockquote",
+          name: "quote",
+          text: "",
+        },
+      ],
```

Comment:
All these vclass elements are being placed, as text, right into the element's class tag. I'm not sure this is what you intended, because it looks like you want this JavaScript to evaluate somewhere. What's this supposed to do?

Reply:
Solved via reverting template to not use v-for